examples/resnet20_cifar10.py

```diff
@@ -2,17 +2,19 @@
 from time import perf_counter
 import numpy as np
 import tensorflow as tf
+
 from ml_genn import Model
 from ml_genn.converters import DataNorm, SpikeNorm, FewSpike, Simple
 from ml_genn.utils import parse_arguments, raster_plot
+from transforms import *
 
 
 batch_size = 256
 epochs = 200
 learning_rate = 0.05
 momentum = 0.9
 
-steps_per_epoch = 50000 // epochs
+steps_per_epoch = 50000 // batch_size
```

Comment:
is this right?

Reply:
Hmm. No.. Not sure how that got in there. Will change it back.